Makefile

```diff
@@ -589,7 +589,7 @@ flexlink.opt:
 	mv flexlink.exe flexlink && \
 	$(MAKE) OCAML_FLEXLINK="../boot/ocamlrun ./flexlink" MSVC_DETECT=0 \
 	           OCAML_CONFIG_FILE=../config/Makefile \
-	           TOOLCHAIN=$(TOOLCHAIN) TOOLPREF=$(TOOLPREF) \
+	           TOOLCHAIN=$(TOOLCHAIN) \
```

Comment:
Sorry, I only reviewed this by the previous diff - good spot to remove `TOOLPREF` here too, but `TOOLCHAIN` can also go.